### PR TITLE
fix: use html-webpack-plugin-for-multihtml to increase compilation speed

### LINKED ISSUE #315: 最新的模板。运行npm start webpack asset optimization  花很长时间
最新的模板。运行npm start webpack asset optimization  花很长时间，并且更新一个文字，感觉会所有文件全部打包。

### PR DESCRIPTION
**Pull request by hacke2**
resolve  https://github.com/weexteam/weex-toolkit/issues/315

If we have many vue component. It will so slow when a minor change.

The reason is that every component of weex will generate a page, and each time HtmlWebpackPlugin is modified, it will analyze whether there is a file to modify.

For example, there are 30 files in my project.

Before optimization:

image

After optimization:

image

Speed increased by 95%!!! 🎉

### CODE REVIEW

**Review by erha19 — CHANGES_REQUESTED** [review]
The `html-webpack-plugin-for-multihtml` module can't be found in the package.json, it will cause an error about `cannot find module 'html-webpack-plugin-for-multihtml'`.

File: template/package.json [hunk 1]
```diff
@@ -85,6 +85,7 @@
     "extract-text-webpack-plugin": "^3.0.2",
     "friendly-errors-webpack-plugin": "^1.6.1",
     "fs-extra": "^5.0.0",
+    "hard-source-webpack-plugin": "^0.12.0",
```
Comment (erha19) [inline template/package.json]: What's this module used for?

Comment (hacke2) [reply template/package.json]: my fault, fixed

**Review by erha19 — APPROVED** [review]

**Comment by weiweiwei256** [comment]
@hacke2   Is There any details for using html-webpack-plugin-fro-multihtml?  Demo code.  
I have same issue.

**Pull request merged**